src/provision.ts

```diff
@@ -16,11 +16,20 @@ export const startProvisioning = async (button: SerialLaunchButton) => {
   if (!port) {
     return;
   }
-
+  let timeout: string | null ;
+  try {
+    if((timeout = button.getAttribute('timeout')) == null){
+      timeout = '1000';
+    }
+    
+  } catch (error) {
```

Comment:
What can raise?

Reply:
Good point. This is just some left over code, vrey much.